src/coreclr/src/binder/clrprivbinderassemblyloadcontext.cpp

```diff
@@ -57,24 +60,35 @@ HRESULT CLRPrivBinderAssemblyLoadContext::BindAssemblyByName(IAssemblyName     *
     SAFE_NEW(pAssemblyName, AssemblyName);
     IF_FAIL_GO(pAssemblyName->Init(pIAssemblyName));
 
-    // When LoadContext needs to resolve an assembly reference, it will go through the following lookup order:
-    //
-    // 1) Lookup the assembly within the LoadContext itself. If assembly is found, use it.
-    // 2) Invoke the LoadContext's Load method implementation. If assembly is found, use it.
-    // 3) Lookup the assembly within TPABinder (except for satellite requests). If assembly is found, use it.
-    // 4) Invoke the LoadContext's ResolveSatelliteAssembly method (for satellite requests). If assembly is found, use it.
-    // 5) Invoke the LoadContext's Resolving event. If assembly is found, use it.
-    // 6) Raise exception.
-    //
-    // This approach enables a LoadContext to override assemblies that have been loaded in TPA context by loading
-    // a different (or even the same!) version.
+    tracer.SetAssemblyName(pAssemblyName);
 
     {
+        // When LoadContext needs to resolve an assembly reference, it will go through the following lookup order:
+        //
+        // 1) Lookup the assembly within the LoadContext itself. If assembly is found, use it.
+        // 2) Invoke the LoadContext's Load method implementation. If assembly is found, use it.
+        // 3) Lookup the assembly within TPABinder (except for satellite requests). If assembly is found, use it.
+        // 4) Invoke the LoadContext's ResolveSatelliteAssembly method (for satellite requests). If assembly is found, use it.
+        // 5) Invoke the LoadContext's Resolving event. If assembly is found, use it.
+        // 6) Raise exception.
+        //
+        // This approach enables a LoadContext to override assemblies that have been loaded in TPA context by loading
+        // a different (or even the same!) version.
+
         // Step 1 - Try to find the assembly within the LoadContext.
+
+        tracer.SetAssemblyName(pAssemblyName);
+        tracer.GoToStage(BinderTracing::ResolutionAttemptedOperation::Stage::FindInLoadContext);
+
         hr = BindAssemblyByNameWorker(pAssemblyName, &pCoreCLRFoundAssembly);
         if ((hr == HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND)) ||
             (hr == FUSION_E_APP_DOMAIN_LOCKED) || (hr == FUSION_E_REF_DEF_MISMATCH))
         {
+            INT_PTR managedAssemblyLoadContext = GetManagedAssemblyLoadContext();
+
+            tracer.SetManagedALC(managedAssemblyLoadContext);
```

Comment:
Does this mean that the FindInLoadContext stage would not indicate which ALC it was using?